style.css

```diff
@@ -0,0 +1,524 @@
+.body {
+  margin: 0;
+  background-color: white;
+}
+
+header {
+  width: 1590px;
+  height: 50px;
+  background-color: white;
+  border-bottom: 1px solid #888;
+  position: fixed;
+  margin: -10px;
+  font-size: 12px;
+  z-index: 1;
+}
+
+.nav-bar {
+  display: flex;
+  width: 100%;
+  position: fixed;
+  justify-content: space-between;
+  overflow: hidden;
+  padding: 10px 10px;
+  align-items: center;
+}
+
+.left-nav a {
+  float: left;
+  display: inline;
+  padding-left: 25px;
+  color: black;
+  text-decoration: none;
+}
+
+.logo {
+  width: 200px;
+  height: 25px;
+}
+
+#logo-image {
+  width: 200px;
+}
+
+.right-nav {
+  float: right;
+  display: inline;
+  padding-right: 15px;
+}
+
+.button {
+  color: white;
+  background-color: rgb(75, 131, 173);
+  border: 1px solid #245f8c;
+  border-radius: 3px;
+  font-weight: 650;
+  font-size: 12px;
+}
+
+.main {
+  width: 50%;
+  margin: auto;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+#h1 {
+  color: black;
+  font-family: nyt-cheltenham, georgia, 'times new roman', times, serif;
+  font-weight: 700;
+  font-style: italic;
+  font-size: 2rem;
+  position: static;
+  width: 100%;
+}
+
+.title {
+  text-align: center;
+  background-color: white;
+  width: 100%;
+  padding: 30px;
+  margin: auto;
+}
+
+.icons-up {
+  width: 600px;
+  height: 32px;
+  position: static;
+  border-top: 1px solid silver;
+  padding: 20px;
+  margin: auto;
+  display: flex;
+}
+
+.icons-up i {
+  padding-left: 20px;
+}
+
+.first-img {
+  display: block;
+  margin: auto;
+  width: 945px;
+  height: 531px;
+}
+
+.desc {
+  font-family: nyt-cheltenham, georgia, 'times new roman', times, serif;
+  font-size: 15px;
```

Comment:
Same `font-family` property to many classes instead of that you can make a separate class for font and use that class wherever needed in HTML. It makes your code cleaner.